graphics/vtk/Portfile

```diff
@@ -45,8 +45,13 @@ depends_lib-append \
     port:lz4 \
     port:netcdf-cxx \
     port:tiff \
+    port:gl2ps \
+    port:libharu \
+    port:libogg \
```

Comment:
It's a nitpick, but please keep the dependencies sorted by name.

Reply:
Thank you for pointing this out.
It should not be fixed.